Calculator.js

```diff
@@ -6,20 +6,20 @@ function add(){
 
 function subtract(){
 	//Implement this
-	
-	
+
+
 }
 
 function multiply(){
 	//Implement this
-	
-	
+
+
 }
 
 function divide(){
 	//Implement this
-	
-	
-	
+	var v1=parseInt(document.getElementById("input1").value);
```

Comment:
add functionality to HTML page by putting divide button inside the same container class as the add and multiply buttons.